plugins/org.yakindu.sct.model.sexec/src/org/yakindu/sct/model/sexec/transformation/StructureMapping.xtend

```diff
@@ -73,16 +73,17 @@ class StructureMapping {
 	 */
 	def dispatch Scope mapScope(ImportScope scope) {
 		val _scope = scope.createScope
-		for (Package imp : scope.imports) {
-			val props = resolver.getImportedElementsOfType(imp, Property)
-			for (Declaration decl : props) {
-				_scope.declarations.add(decl.createImportDeclaration)
-			}
-			val operations = resolver.getImportedElementsOfType(imp, Operation) 
-			for (Declaration decl : operations) {
-				_scope.declarations.add(decl.createImportDeclaration)
-			}
-		}
+		//FIXME
```

Comment:
@andreasmuelder how to fix this? 

i can use IPackageImport2URIMapper to get PackageImports - is it needed to get a Package for it?